servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -118,6 +118,9 @@ public InodeFile build() {
 
   private boolean mCompleted;
 
+  /** if any block of the file is lost */
+  private boolean mLost;
```

Comment:
I think the lost attribute here is a bit strange, since it is not describing the actual file information, but rather the state of the system. For example, given a file in Tachyon, the entire system could be idle, but a worker died, and technically the file is lost. However, it will never be reflected in the inode, which seems misleading.

I haven't thought about it much, but isLost seems like something that should be computed on demand, and not stored in the inode. What are your thoughts?